docs/en_US/Compression/Pruner.rst

```diff
@@ -471,7 +416,12 @@ PyTorch code
 
    pruner.update_epoch(epoch)
 
-You can view :githublink:`example <examples/model_compress/pruning/model_prune_torch.py>` for more information.
+You can view :githublink:`mnist example <examples/model_compress/pruning/naive_example_torch.py>` for a quick start.
```

Comment:
it is a little strange, why mention quick start here?

Reply:
I want to show the simplest example for the usage of pruners. 
I will refactor the `QuickStart.rst`. Remove the quick start in this file.